src/components/atoms/CategoryTitle/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+
+interface ICategoryTitle {
+  categoryTitle: string;
+}
+
+const CategoryTitle: FC<ICategoryTitle> = ({ categoryTitle }) => (
```

Comment:
no need to separate this component
better create component  like this 
<section>
      <CategoryTitle categoryTitle="Upcoming" />
      <Slider moviesList={upcomingMovies} />
    </section>